CorsixTH/Lua/languages/polish.lua

```diff
@@ -2196,9 +2209,67 @@ options_window = {
   cancel = "Anuluj",
   data_location = "Lokalizacja danych",
   font_location = "Lokalizacja czcionki",
-  language = "Game language",
+  language = "Język gry",
```

Comment:
Good idea. 

I'll leave the final decision to @TheCycoONE though, as it is something that would be useful for other languages too.

...

but on the other hand - the button on which this test is displayed might be to narrow for such long text. I can check it later and attach screenshot.